doc/classes/EditorExportPlugin.xml

```diff
@@ -81,7 +84,7 @@
 			<param index="1" name="type" type="String" />
 			<param index="2" name="features" type="PackedStringArray" />
 			<description>
-				Virtual method to be overridden by the user. Called for each exported file, providing arguments that can be used to identify the file. [param path] is the path of the file, [param type] is the [Resource] represented by the file (e.g. [PackedScene]) and [param features] is the list of features for the export.
+				Virtual method to be overridden by the user. Called for each exported file, except for imported resources (resources that have an [code].import[/code] file). The arguments that can be used to identify the file. [param path] is the path of the file, [param type] is the [Resource] represented by the file (e.g. [PackedScene]) and [param features] is the list of features for the export.
```

Comment:
Might be worth to mention how to handle `.import` resources if they don't appear in this method.
Also I see the description mentions PackedScene, but isn't it handled by `_customize_scene`? (idk if it applies to all scenes or only imported scenes)

Reply:
`PackedScene` `.tscn` files (which do not have an `.import` file and are not an "imported" resource) are handled by both `_export_file` and `_customize_scene`. So the original description is correct, and maybe even reasonable to highlight that they are handled by `_export_file` in spite of `_customize_scene` also handling them.

Good point about mentioning how to handle resources.